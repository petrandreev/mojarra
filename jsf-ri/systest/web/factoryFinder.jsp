<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <title>Print out the names of the current Factories</title>
    <%@ taglib uri="http://java.sun.com/jsf/core"  prefix="f" %>
    <%@ taglib uri="http://java.sun.com/jsf/html"  prefix="h" %>
  </head>

  <body>
    <h1>Print out the names of the current Factories</h1>

<f:view>

  <h:form>

      <h:outputText value="#{test3.factoryPrintout}" />

  </h:form>

</f:view>

    <hr>
  </body>
</html>
