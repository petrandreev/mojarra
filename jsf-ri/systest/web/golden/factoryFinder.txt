<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <title>Print out the names of the current Factories</title>
    
    
  </head>

  <body>
    <h1>Print out the names of the current Factories</h1>

<form id="_id1" method="post" action="/jsf-systest/faces/factoryFinder.jsp;jsessionid=9F5D0333D14A581AA4BCC739ABE3902C" enctype="application/x-www-form-urlencoded">
ApplicationFactoryWrapper FacesContextFactoryWrapper LifecycleFactoryWrapper RenderKitFactoryWrapper <input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="_id2" /><input type="hidden" name="_id1" value="_id1" /></form>

    <hr>
  </body>
</html>
